docs/Sprint-Review/sprint-38-summary.md

```diff
@@ -0,0 +1,62 @@
+# Sprint 38 Summary
+** 1/5/2021 - 1/18/2021 **
+
+## Sprint 38 Goals
+
+* Documentation (reorganizing directories)
+* Resubmission research synthesis
+* ACF AMS
+
+## Closed/Merged/Completed (Done/Demo, Closed)
+- [#1136](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1136) - [Backend] Accept AMS OpenId
+- [#1520](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1520) - As technical lead, I need ACF AMS integrated for direct DAC access
+- [#1525](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1525) - Update TANF-app/doc/Sprint-Review/
+- [#1524](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1524) - Update TANF-app/doc/Security-Compliance/
+- [#573](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/573) - As a dev, I want a README for the /scripts folder that describes what each script is for and how it is used in the CI process.
+- [#1082](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1082) - Update ADRs
+- [#1483](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1483) - Update Current TDRS documentation
+- [#1482](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1482) - Update workflow.md
+
+## Submitted (QASP Review, OCIO Review)
+- [#1346](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1346) - [SPIKE] Test hero images in code & adjust spec accordingly
+- [#1418](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1418) - [Research] Code docs synthesis of Resubmission Research
+- [#1522](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1522) - Update TANF-app/doc/Product-Strategy/
+- [#1521](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1521) - Update TANF-app/doc/Background/
+- [#1523](https://app.zenhub.com/workspaces/tdrs-sprint-board-5f18ab06dfd91c000f7e682e/issues/raft-tech/tanf-app/1523) - Update TANF-app/doc/How-We-Work/
```

Comment:
at least a few of these can be moved to the done section if they get merged today. are any of these still in qasp? @valcollignon @lfrohlich

Reply:
Good catch! Product Strategy and How we work are just waiting to be merged. Background was merged. @valcollignon  can you update the summary accordingly?

Reply:
Done!